### PR TITLE
Email notifications

### LINKED ISSUE #98: Fix seeds file to be idempotent

### PR DESCRIPTION
**Pull request by teknofire**
Sets up a new worker, siteconfig settings that will need to be loaded

Updated seeds file to be idempotent, closes #98

Added rake task to add in the default siteconfig: `rake sandy:update:siteconfig RAILS_ENV=production`
Another rake task to run to setup default notifications: `rake db:seed sandy:update:notifications`

- [x] rake task to setup siteconfig default values
- [x] Setup system for triggering when emails need to happen
- [x] Setup system for allowing users to select what emails they want to receive

### CODE REVIEW

**Comment by teknofire** [comment]
@smacfarlane ready to review

**Comment by teknofire** [comment]
just realized something else that I need to add to this

**Comment by teknofire** [comment]
@smacfarlane ok this is now ready to go

File: lib/tasks/sandy.rake [hunk 1]
```diff
@@ -1,5 +1,24 @@
 namespace :sandy do
   namespace :update do
+    desc "Inject the siteconfig values"
+    task siteconfig: :environment do
+      unless SiteConfig.first.nil?
+        SiteConfig.create(name: 'GINA::NRT Dashboard', host: 'http://nrt-dashboard.gina.alaska.edu')
```
Comment (smacfarlane) [inline lib/tasks/sandy.rake]: host should be nrt-status.gina.alaska.edu,  or we need to change the domain name.

**Comment by smacfarlane** [comment]
Need to add mailer config for production, then this looks good to go.

**Pull request merged**